aodncore/pipeline/handlerbase.py

```diff
@@ -641,6 +639,7 @@ def _archive(self):
             self._upload_runner_archive.run(files_to_archive)
 
         if self.archive_input_file:
+            self.input_file_object.publish_type = PipelineFilePublishType.ARCHIVE_ONLY
```

Comment:
I think this is ok, except in the (unlikely?) case I want to e.g. archive _and harvest_ the input file. I can still do that, just not using the `archive_input_file`, because if I set that this will then clobber the actual publish_type I want (set elsewhere).

I guess we could add something like 
```python
if self.input_file_object.publish_type is PipelineFilePublishType.UNSET
```